File: Modul 4/Soal 4/PRAK304-2210817110002-MUHAMMAD PASHA NABEEL.c
#include <stdio.h>

int main()
{
    int n;
    scanf("%d", &n);
    if(n==0){
        printf("Nol\n");
    }
    else if(n<=9){
        printf("Satuan\n");
    }
    else if(n>=100){
        printf("Anda Menginput Melebihi Limit Bilangan\n");
    }
    else if(n>=11&&n<=19){
        printf("Belasan\n");
    }
    else{
        printf("Puluhan\n\n");
    }
    return 0;
}

File: Modul 4/Soal 4/PRAK304-2210817110002-MUHAMMAD PASHA NABEEL.py
nilai = int(input())

if nilai == 0:
    print("Nol")
elif nilai <= 9:
    print("Satuan")
elif nilai >=100:
    print("Anda Menginput Melebihi Limit")
elif 11 <= nilai <= 19:
    print("Belasan")
else:
    print("Puluhan")